sphinx_math_dollar/tests/test_extension.py

```diff
@@ -21,8 +23,4 @@ def _test_sphinx_build(app, status, warning):
     assert r"\[math\]" not in html
     assert r"\[nomath\]" not in html
 
-    assert not status.read()
```

Comment:
Is this checked automatically by the new fixture?

Reply:
No idea... The only documentation I had was this issue regarding that there is no documentation... https://github.com/sphinx-doc/sphinx/issues/7008

Reply:
Or are status and warning pytest fixtures? ( I have really no idea how this works, but, maybe, starting to pick up some things in hind-sight...)

Reply:
Seems like I learnt a bit today... Had so much work trying to get it to work so forgot about them. But they are back now.